src/utilities/idf/WorkspaceObject.cpp

```diff
@@ -422,10 +422,16 @@ namespace detail {
 
   // Pre-condition:  Object valid at Workspace's strictness level.
   bool WorkspaceObject_Impl::setString(unsigned index, const std::string& value, bool checkValidity) {
+    StrictnessLevel level = m_workspace->strictnessLevel();
+
+    // StrictnessLevel::None
+    if (level < StrictnessLevel::Minimal) {
+      checkValidity = false;
+    }
```

Comment:
Not a big fan of simply overriding the arg, but that's mostly cosmetic at this point, but maybe I'll feel even more strongly about this once I've looked at all the changes.

Reply:
The alternative is to do something like that on line 461 below for eg:

```diff
-      if (checkValidity) {
+      if (checkValidity && level > StrictnessLevel::None) {
         if (!fieldDataIsValid(index, level).empty()) {
```

Reply:
Function calls like `convertToTargetHandle(value, iddObject().objectLists(index), checkValidity);` would (or should after modification) work because they do stuff like this already:

```
    if ((!checkValidity) || (m_workspace->strictnessLevel() < StrictnessLevel::Draft)) {
```